src/components/CardList.tsx

```diff
@@ -54,7 +54,7 @@ const Card = (props: CardListItem & Omit<StackProps, "title" | "id">) => {
 
   return (
     <CardContainer {...rest}>
-      {image && <Image src={image} alt={alt ?? ""} minW="20px" />}
+      {image && <Image src={image} alt={alt ?? ""} w="20px" />}
```

Comment:
The short `w` prop is a chakra prop, is only a visual change, just css, and is not caught by Next.

Since what we really want here is a fixed width image, we can use the `width' prop to tell Next Image that we do not need a larger image. Then Next will only serve a 20px image. Instead of serving a larger image and shrinking it in the browser.

```suggestion
      {image && <Image src={image} alt={alt ?? ""} width={20} />}
```

Reply:
Ah gotcha, hadn't realized the difference getween `w` and `width` with this component, thanks.. (`w={20}` was interpreted as 5 rem by Chakra)